Hestia/Hestia/frontend/Firebase/ServerSelectScreen/TableSourceServerList.cs

```diff
@@ -28,23 +28,23 @@ public override nint NumberOfSections(UITableView tableView)
 
 		public override nint RowsInSection(UITableView tableview, nint section)
 		{
-            return Globals.FirebaseServers.Count;
+            return Globals.Auth0Servers.Count;
```

Comment:
Indentation here looks awful, but I think that's just on github.

Reply:
Was also not visible in Visual Studio, but should be fixed in e5e31a2aabbf4b7159da381fbe22dbdba1baf630